### ISSUE TITLE
Bandit error: no attribute 'NameConstant'

### ISSUE THREAD

**Issue by nnadeau** [nnadeau/pybotics#127]
## Expected Behavior
- `bandit` linting works without error

## Actual Behavior
- [CI passes...](https://travis-ci.org/nnadeau/pybotics/jobs/254838889)
- `bandit` throws an error on Python 3.3:
```
ERROR	Bandit internal error running: start_process_with_partial_path on file pybotics/robot.py at line 385: 'module' object has no attribute 'NameConstant'Traceback (most recent call last):
```

## Steps to Reproduce the Problem
- See https://travis-ci.org/nnadeau/pybotics/jobs/254838889

## Specifications
- Pybotics Version: 0.3.2
- Python Version: 3.3
- Other Package Versions: `bandit-1.4.0`

**Comment by nnadeau** [comment]
- Most likely won't fix as Python 3.3 is on its way to deprecation
- Python 3.3 will stay in `allowed failures` in Travis CI

**Comment by nnadeau** [comment]
Similar: https://bugs.launchpad.net/bandit/+bug/1613431
